install_config/certificate_customization.adoc

```diff
@@ -95,3 +95,6 @@ Relative paths are resolved relative to the master configuration file. Restart
 the server to pick up the configuration changes.
 
 For the master API or web console, wildcard names are accepted.
+
+The provided named certificate can also be added using the 
+xref:../install_config/redeploying_certificates.adoc#redeploying-new-custom-ca[*_redeploy-certificates.yml_* playbook]. 
```

Comment:
The `redeploy-certificates.yml` is pointing to wrong section. Should be the following one:
https://github.com/mburke5678/openshift-docs/blob/f2d4201f580ba505ee061f5224ffadb5abe7c896/install_config/redeploying_certificates.adoc#redeploying-all-certificates-using-the-current-product-title-and-etcd-ca